images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -120,6 +120,15 @@ function migrate_db() {
   ) 2>&1 | tee ${PV_MIGRATE_DB_LOG}
 }
 
+# Set EVM admin pwd
+function set_admin_pwd() {
+ echo "== Setting admin password =="
+
+   cd ${APP_ROOT} && bin/rails runner -e production "EvmDatabase.seed_primordial; user = User.find_by_userid('admin'); user.password = ENV['APPLICATION_ADMIN_PASSWORD']; user.save; exit;"
```

Comment:
I think we can simplify the ruby here.

```ruby
EvmDatabase.seed_primordial; User.find_by_userid('admin').update_attributes!(:password => ENV['APPLICATION_ADMIN_PASSWORD'])
```

@bdunne this look good to you?

Reply:
Also, I think `RAILS_ENV=production` is exported in `/etc/default/evm` so we shouldn't need the `-e production` here.

Reply:
@carbonin Yep , will go ahead and test with the simplied version. `/etc/default/evm` is  already sourced so should work without -e.